src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import '@webcomponents/webcomponentsjs';
```

Comment:
delete please

Reply:
Are you sure? This is setup by the scaffolding script. Why don't we need to import it? Where is it injected?

Reply:
We have a separate @axa-ch/polyfill package that includes it. Consumers must include it. It comes with inconvenient side effects if we include them in our packages.